modules/modules.nix

```
@@ -120,6 +120,7 @@ let
     (loadModule ./services/redshift.nix { })
     (loadModule ./services/rsibreak.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/screen-locker.nix { })
+    (loadModule ./services/slack.nix { })
```

The module should be limited to Linux since it uses systemd
```suggestion
    (loadModule ./services/slack.nix { condition = hostPlatform.isLinux; })
```